sdk/python/dagger/src/dagger/engine.py

```diff
@@ -8,11 +9,11 @@ class Engine:
 
     def __init__(self,
                  port: int = 8080,
-                 workdir: str = None,
-                 configPath: str = None):
```

Comment:
nit: I think it's more idiomatic to keep `None`. You can make it optional like this:

```suggestion
                 workdir: str | None = None,
                 configPath: str | None = None):
```

Reply:
For previous versions you'd need to do this:
```python
from typing import Optional
...
workdir: Optional[str] = None
```